Bottom: 85605fba8953394ce0cd60fc88023eda76b12ce9
Top:    6cf066baacd8cc6bf4b7ad5b4491d381a1105a84
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 14:43:07 +0100

Shiny 3D eye-candy

Experiments in progress - don't expect this to work yet!


---

diff --git a/src/hid/gtk/gui-pinout-preview.c b/src/hid/gtk/gui-pinout-preview.c
index 89a3563..fbb89de 100644
--- a/src/hid/gtk/gui-pinout-preview.c
+++ b/src/hid/gtk/gui-pinout-preview.c
@@ -211,7 +211,7 @@ ghid_pinout_preview_expose (GtkWidget * widget, GdkEventExpose * ev)
   glOrtho (0, widget->allocation.width, widget->allocation.height, 0, -100000, 100000);
   glMatrixMode (GL_MODELVIEW);
   glLoadIdentity ();
-  glTranslatef (0.0f, 0.0f, -Z_NEAR);
+//  glTranslatef (0.0f, 0.0f, -Z_NEAR);
 
   glClearColor (gport->bg_color.red / 65535.,
                 gport->bg_color.green / 65535.,
